src/nvidia-cuda/README.md

```diff
@@ -0,0 +1,29 @@
+
+# NVIDIA CUDA (nvidia-cuda)
+
+Installs shared libraries for NVIDIA CUDA.
+
+## Example Usage
+
+```json
+"features": {
+        "ghcr.io/devcontainers/features/nvidia-cuda:1": {
+            "version": "latest"
+        }
+}
+```
+
+## Options
+
+| Options Id | Description | Type | Default Value |
+|-----|-----|-----|-----|
+| installCudnn | Additionally install CUDA Deep Neural Network (cuDNN) shared library | boolean | - |
+| installNvtx | Additionally install NVIDIA Tools Extension (NVTX) | boolean | - |
+| cudaVersion | Version of CUDA to install | string | 11.7 |
+| cudnnVersion | Version of cuDNN to install | string | 8.5.0.96 |
+
+
+
+---
+
+_Note: This file was auto-generated from the [devcontainer-feature.json](https://github.com/devcontainers/features/blob/main/src/nvidia-cuda/devcontainer-feature.json).  Add additional notes to a `NOTES.md`._
```

Comment:
@jungaretti If there's any _additional_ docs we should add into this auto-generated readme, you can do so with a `NOTES.md` file in the `src/nvidia-cuda` directory.  It will be auto-inserted into this auto-generated doc each time (other features have an example of this).